packages/eslint-plugin-calcite-components/src/rules/enforce-ref-last-prop.ts

```diff
@@ -17,20 +17,37 @@ const rule: Rule.RuleModule = {
       JSXIdentifier(node) {
         const openingElement = node.parent as JSXOpeningElement;
         if (openingElement.type === "JSXOpeningElement") {
-          const attributes: string[] = [];
+          const attributes: (JSXAttribute | JSXSpreadAttribute)[] = [];
 
           openingElement.attributes.forEach((attr: JSXAttribute | JSXSpreadAttribute) => {
             if (attr.type === "JSXAttribute" && attr.name?.type === "JSXIdentifier") {
-              attributes.push(attr.name.name);
+              attributes.push(attr);
             }
           });
 
-          const refAttribute = attributes.find((attr: string) => attr === "ref");
+          const refAttribute = attributes.find(
+            (attr: JSXAttribute | JSXSpreadAttribute) =>
+              attr.type === "JSXAttribute" && attr.name?.type === "JSXIdentifier" && attr.name.name === "ref",
+          );
 
           if (refAttribute && attributes.indexOf(refAttribute) !== attributes.length - 1) {
             context.report({
               node,
               message: `"ref" prop should be placed last in JSX to ensure the node attrs/props are in sync.`,
+              fix(fixer) {
```

Comment:
Can we update tests to cover the fixer (this is new territory for me, but I think we can follow what [`stencil-eslint` does](https://github.com/stencil-community/stencil-eslint/blob/425cfa20695d56d197641b2ec3c24ecbc3f5d6bd/tests/lib/rules/own-props-must-be-private/own-props-must-be-private.spec.ts#L10))?